model/schema/variation.yaml

```diff
@@ -118,11 +135,8 @@ classes:
       object:
         required: true
 
-
   VariantTranscriptLocationAssociation:
```

Comment:
Now that object has range Transcript, it looks like there's no place to store the Chromosome (it had been the range of object until this level down in the hierarchy). Maybe we need another tag for chromosome of origin? There is a 'chromosome_curie' slot for the VariantLocationAssociationDTO class, but I don't think there is an equivalent chromosome slot in VariantTranscriptLocationAssociation or VariantPolypeptideLocationAssociation

Reply:
@chris-grove I don't think we would need it.  Presumably a transcript would be associated with a gene, which would be located on a chromosome, so this doesn't need to be submitted for the VariantTranscriptLocationAssociation.  I would assume that similarly a protein would be associated with a transcript object....